src/Components/Schedules/NewSchedule.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import Box from "@mui/material/Box";
 import Grid from "@mui/material/Grid";
-import Typography from "@mui/material/Typography";
+import { Button } from "@mui/material";
```

Comment:
Do something consistent with MUI imports - either import them one by one like you're doing here, or import them altogether like you're doing on line 7.